eva-accession-ws/src/test/java/uk/ac/ebi/eva/accession/ws/SubmittedVariantsRestControllerTest.java

```diff
@@ -503,4 +504,25 @@ public void testGetDeprecatedDbsnpSubmittedVariant()
         assertEquals(variant1, response.getBody().get(0).getData());
         assertCreatedDateNotNull(response.getBody());
     }
+
+    @Test
+    public void testGetNonExistentDbsnpSubmittedVariant()
```

Comment:
If you remove the changes in the DbsnpSubmittedVariantAccessioningDatabaseService.java  class, you'll see that the test passes, so the test is not working. Without looking into it, my bet is the mocked classes.

This kind of bugs are the best ones to do TDD, writing the tests before the fix to see it failing and make it pass.

Reply:
wait, I think I did something wrong with maven, now I'm not sure that the test passes without the fix, let me confirm.

Reply:
yes, confirmed the test passes without the fix. please check.